src/module/AddRemove.js

```diff
@@ -0,0 +1,124 @@
+const todoListContianer = document.querySelector('.todo__list');
+const listConatiner = document.querySelector('.list__container');
+const singleList = document.createElement('li');
+export const clearComoleted = document.querySelector('.clearMarked');
+export const inputTodo = document.querySelector('.add__list');
+
+export default class TodoList {
+  constructor() {
+    this.list = localStorage.getItem('todoList')
+      ? JSON.parse(localStorage.getItem('todoList'))
+      : [];
+  }
+
+  addTodo() {
+    const listLength = this.list.length;
+    if (inputTodo.value !== '') {
+      this.list.push({
+        description: inputTodo.value,
+        completed: false,
+        index: listLength,
+      });
+    }
+
+    this.reArrange();
+    localStorage.setItem('todoList', JSON.stringify(this.list));
+    inputTodo.value = '';
+  }
+
+  markList(content, index, marked) {
+    if (marked === true) {
+      this.list[index].completed = true;
+      const p = content.parentNode.querySelector('.tagP');
+      p.classList.add('strike');
+    } else {
+      this.list[index].completed = false;
+      const p = content.parentNode.querySelector('.tagP');
+      p.classList.remove('strike');
+    }
+    const hold = this.list.filter((item) => item);
+    this.list = hold;
+    localStorage.setItem('todoList', JSON.stringify(this.list));
+  }
+
+  removeList() {
+    const strickers = document.querySelectorAll('.strike');
+    strickers.forEach((value) => {
+      const parentContainerLi = value.parentNode.parentNode;
+      parentContainerLi.style.display = 'none';
+      const title = parentContainerLi.querySelector('.tagP').textContent;
+      this.list = this.list.filter((value) => value.description !== title);
+    });
+
+    this.reArrange();
+    localStorage.setItem('todoList', JSON.stringify(this.list));
+  }
+
+  // SMALL UTILITIES
+
+  editListWrite(pDots, index) {
+    this.list[index].description = pDots.innerText;
+    localStorage.setItem('todoList', JSON.stringify(this.list));
+  }
+
+  reArrange() {
+    this.list.forEach((value, index) => {
+      value.index = index + 1;
+    });
+  }
+}
+
+export function render(member) {
+  todoListContianer.innerHTML = '';
+  member.forEach((list) => {
+    const check = list.completed ? 'checked' : null;
+    singleList.innerHTML = `
+    <div class="check">
+      <input type="checkbox" name="completed" value="${list.index}" ${check}  class="chBox"/>
```

Comment:
- [ ][OPTIONAL] I was expecting to see the completed state of a task changes when the check box got triggered. Can you please check on that too? I think you might need an onChangeEventListener for that. Please take a look at this answer in [stackoverflow](https://stackoverflow.com/questions/14544104/checkbox-check-event-listener) for a guide.